### ISSUE TITLE
6.4.8 - select known models stopped working for Lenovo, works for Dell/HP

### ISSUE THREAD

**Issue by equillibrium** [maurice-daly/DriverAutomationTool#79]

**Comment by rtruss** [comment]
ditto

**Comment by viggenz** [comment]
Fresh install of 6.4.8 and the Lenovo option is greyed out on the Make & model selection > Manufacture Selection

**Comment by matthewv87** [comment]
With over 40 Lenovos in my environment I'd love for this feature to work again. Currently getting this error:
```
Exception calling "Substring" with "2" argument(s): "Index and length must refer to a location within the string.
Parameter name: length"
At line:15231 char:64
+ ... Find-LenovoModelType -ModelType $($LenovoKnownModel.Substring(0, 4)))Exception calling "Substring" with "2" argument(s): "Index and length must refer to a location within the string.
Parameter name: length"
At line:15231 char:64
+ ... Find-LenovoModelType -ModelType $($LenovoKnownModel.Substring(0, 4)))
```

**Comment by MJPhelan** [comment]
In 6.49 - If you are using Server 2016 - install this - https://visualstudio.microsoft.com/vs/older-downloads/isolated-shell/

**Comment by rtruss** [comment]
I was getting that on windows 10 1909 too.  looks to be corrected in 6.49 however.

**Comment by maurice-daly** [comment]
Version 6.4.9 contains updates for model matching.
